### ISSUE TITLE
Not compilable with VS2015

### ISSUE THREAD

**Issue by Dokman** [callmephil/SunWellCore#22]
Like says the title it cannot compile with VS2015 at x64 in x86 i didn't test it

**Comment by lashia** [comment]
https://github.com/Kittnz/Sunwell/commit/02432d276d6c7cd05c4ae74792ced6b61b17abe7
can you test this?

**Comment by Dokman** [comment]
yes and it's working i thought, i have to test the runnables but the mmaps generator doesn't compile MapManager.cpp at Line 722 and 723 there are arguments that are not public i think it's from recastnavigation

**Comment by callmephil** [comment]
https://github.com/callmephil/SunWellCore/commit/a61c72d84eaf72b76242a1995d0d511a914277d6 but i do not recommend to use extractors download mmaps/dbc/maps/vmaps from www.sunwell.pl

**Comment by Dokman** [comment]
but you can fix it?

**Comment by callmephil** [comment]
i linked the commit at first on my previous comment. but since they have custom mmaps i can't really fix the error just disable it so you can compile with extractor unless i read all the code they made for it so i can understand properly how it works.
